hpx/components/containers/partitioned_vector/partitioned_vector_component.hpp

```diff
@@ -124,6 +124,12 @@ namespace hpx { namespace server
         }
 
         ///////////////////////////////////////////////////////////////////////
+        void set_data(data_type other)
+        {
+            partition_vector_ = other;
+        }
+
```

Comment:
I'd suggest to change this to:

    void set_data(data_type && data) 
    {
        partition_vector_ = std::move(data);
    }

to avoid extra copies of the whole partition.